flow/src/main/java/com/sequenceiq/flow/core/Flow2Handler.java

```diff
@@ -413,8 +413,7 @@ private boolean isAcceptablePayload(Payload payload) {
                 && !((Acceptable) payload).accepted().isComplete();
     }
 
-    public FlowIdentifier retryLastFailedFlow(Long resourceId, java.util.function.Consumer<FlowLog> beforeRestart) {
-        List<FlowLog> flowLogs = flowLogService.findAllForLastFlowIdByResourceIdOrderByCreatedDesc(resourceId);
+    private void canRetryBePerformed(List<FlowLog> flowLogs) {
```

Comment:
From the name here, I'd expect a boolean return value
checkRetryCanBePreformed or AssertRetryCanBePreformed

Reply:
Or checkRetryable(flowLogs)

Reply:
will update.